hive/thrift-metastore.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+source /usr/lib/hustler/bin/qubole-bash-lib.sh
+
+is_master=$(nodeinfo is_master)
```

Comment:
As https://github.com/qubole/bootstrap-functions/blob/master/common/utils.sh is now available, we should use that, and create new methods for thrift_metastore, hive_version